item.py

```diff
@@ -54,12 +54,16 @@ def parseItem(m, compendium, args):
             typ.text = 'G'
         if m['type'] == "AT":
             typ.text = 'G'
+        if m['type'] == "FD":
+            typ.text = 'G'
         if m['type'] == "GS":
             typ.text = 'G'
         if m['type'] == "INS":
             typ.text = 'G'
         if m['type'] == "MNT":
             typ.text = 'G'
+        if m['type'] == "MR":
+            typ.text = 'W'
         if m['type'] == "OTH":
             typ.text = 'G'
```

Comment:
These must be new types, I don't recall seeing them before.

Reply:
For stuff like this, I like to add the type name to the header in the section starting at line 134

Reply:
Added "Food and Drink" for "FD".
"MR" isn't entirely apparent, but considering its source is "UAPrestigeClassesRunMagic" [sic] I've added "Magic Rune"